libr/bin/mangling/swift-sd.c

```diff
@@ -217,8 +234,6 @@ R_API char *r_bin_demangle_swift(const char *s, bool syscmd, bool trylib) {
 		}
 	}
 
```

Comment:
```suggestion
		out[0] = 0;
```

Reply:
no since `RStrBuf *out` is not a char array